Modules/timemodule.c

```diff
@@ -1421,13 +1421,15 @@ PyInit_timezone(PyObject *m) {
 #if defined(HAVE_TZNAME) && !defined(__GLIBC__) && !defined(__CYGWIN__)
     PyObject *otz0, *otz1;
     tzset();
+#ifndef __VXWORKS__
     PyModule_AddIntConstant(m, "timezone", timezone);
```

Comment:
Does `timezone` not exist? What about `_timezone`?

Reply:
No timezone, no _timezone,  you have to use tzset/tzget